test-resources/include-json-ld.html

```diff
@@ -0,0 +1,35 @@
+<!--
+Copyright 2017 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not
+use this file except in compliance with the License. You may obtain a copy of
+the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+License for the specific language governing permissions and limitations under
+the License.
+-->
+<script src="inject-element-after-load.js"></script>
+<link rel="import" href="custom-element.html">
```

Comment:
Can we add the other `<script>`s that you have listed in your other comment to the test?

Reply:
Done